src/Microsoft.AspNetCore.SignalR.Client/Utils.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    internal static class Utils
```

Comment:
Urg, no. Also we might be able to use the types in WebUtilities for this.

/cc @Tratcher